src/lang/uk.rs

```diff
@@ -399,162 +393,271 @@ const KOPIYKAS: [[&str; 6]; 2] = [
+            ),
+            Currency::JPY => format!(
+                "сен{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::KRW => format!(
+                "чон{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::KZT => format!(
+                "тиїн{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::MYR => format!(
+                "сен{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
             Currency::NOK => String::from("оре"),
             Currency::PEN => String::from("сентімо"),
-            Currency::PLN => format!("грош{}", NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declenation_idx]),
-            Currency::QAR | 
-            Currency::RIYAL |
-            Currency::SAR => format!("філс{}", NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declenation_idx]),
-            Currency::RUB => //format!("копійк{}", NOUN_1ST_GROUP_FEMININE_HARD_DECLENSIONS[number_idx][declenation_idx]),
-                             String::from("руській воєнний корабль іді нахуй"),
-            Currency::THB => format!("cатанг{}", NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declenation_idx]),
-            Currency::TRY => format!("лір{}",NOUN_1ST_GROUP_HARD_DECLENSIONS[number_idx][declenation_idx]),
-            Currency::UAH => String::from(KOPIYKAS[number_idx][declenation_idx]),
+            Currency::PLN => format!(
+                "грош{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::QAR | Currency::RIYAL | Currency::SAR => format!(
+                "філс{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::RUB =>
+            //format!("копійк{}", NOUN_1ST_GROUP_FEMININE_HARD_DECLENSIONS[number_idx][declination_idx]),
+            {
+                String::from("руській воєнний корабль іді нахуй")
+            }
+            Currency::THB => format!(
+                "cатанг{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::TRY => format!(
+                "лір{}",
+                NOUN_1ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::UAH => String::from(KOPIYKAS[number_idx][declination_idx]),
             Currency::VND => String::from("су"),
-            Currency::ZAR => format!("цент{}", NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declenation_idx]),
+            Currency::ZAR => format!(
+                "цент{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
             //_ => currency.default_cent_string(self.number == GrammaticalNumber::Plural)
```

Comment:
ditto my comment above

Reply:
Pavlo, you're working late once again...